sig-multicluster/README.md

```diff
@@ -24,17 +25,59 @@ A Special Interest Group focussed on solving common challenges related to the  m
+| Owner(s) | Working Group |
+| Where to find it? | Code prototypes in development (one example here). Summary of meetings notes and meeting times [available](ttps://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY). |
+
+## Cluster Registry
+Common abstraction for a Registry of Clusters that can store per-Cluster metadata and supports Kubernetes label selection. The Cluster Registry can be built as a standalone or an aggregated API server and currently provides a Registry of Clusters without any actively reconciling Kubernetes controller. The API design is documented here and is intended to serve as a basis to develop multicluster controllers.
+
+|  |  |
+| -------------- |:-------|
+| Recommended for Production | Not yet. |
+| General Availability | Beta by 2018-Q1, GA by 2018-Q2 (see Project plan for details). |
+| Current Level of Activity | Alpha, active development for Beta and GA milestones. |
+| Owner(s) | [@perotinus](https://github.com/perotinus), [@ifont](https://github.com/ifont) |
+| Where to find it? | https://github.com/kubernetes/cluster-registry |
+
+## Kubemci (Kube Multi-cluster Ingress)
+Ability to program Global multicluster load balancers with two deliverables:
+* Standalone CLI tool without a centralized control plane. Allows ingress configurations that span multiple clusters. It needs to be invoked whenever there is a change in the number of clusters (healthchecking is still maintained in the background). Much of this work will be folded back into the next deliverable.
+* Kubernetes style multicluster API and self-healing controller with cluster registry
+
+|  |  |
+| -------------- |:-------|
+| Recommended for Production | Not yet. |
+| General Availability | CLI: Beta by 2018-Q1, Controller: Alpha by 2018-Q1 |
+| Current Level of Activity? | CLI is alpha and team is looking for feedback. Active development is on to bring it to beta. |
+| Owner(s) | [@nikhiljindal](https://github.com/nikhiljindal), [@G-Harmon](https://github.com/G-Harmon) |
+| Where to find it? | https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress |
+
+
+
+## Federation v1 (a.k.a. Cluster Federation)
+Control Plane that Federates a subset of existing single-cluster Kubernetes API across clusters. Initially released as part of Kubernetes 1.3, Cluster Federation has implemented parts of the existing “single-cluster” Kubernetes API so that they may be used transparently in a multi-cluster fashion. As of Kubernetes 1.9, many of the non-Federated Kubernetes Workloads APIs have moved to GA (Deployments, ReplicaSets, Daemonset, StatefulSet). However, under Federation, these workloads have not reached the maturity of “GA” since they are re-implemented by different federated controllers. The SIG is currently re-thinking how to support Kubernetes APIs in a Federation fashion and this discussion is reflected under Federation v2.
```

Comment:
Consider s/Federates/federates/ and defining what 'federates' means (e.g. coordinate configuration across multiple clusters).